src/picknik_ur_site_config/objectives/get_fixed_handle_graspable_object.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="GetFixedHandleGraspableObject">
+  <!-- ////////// -->
+  <!-- TODO: need to be directly in front of handle for this behavior to work until (https://github.com/PickNikRobotics/moveit_studio/issues/5968) is fixed or mesh completion behaviors are completed. -->
```

Comment:
Don't link to private repo issues in this workspace, keeping it more generic is OK